app/off.py

```diff
@@ -1,4 +1,5 @@
 from urllib.parse import urljoin
+from wikidata.client import Client
```

Comment:
I would better use 
```pythof
import wikidata.client
```

and latter on

```python
    client = wikidata.client.Client()
```

I think it's worth it because, it hard otherwise to remember we are talking about wikidata.